use_cases/eluc/experiments/predictor_experiments.ipynb

```diff
@@ -203,12 +216,13 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "MAE Random Forest: 0.037707034150731636\n"
+      "MAE Random Forest: 0.03978765055906081\n"
      ]
     }
    ],
    "source": [
-    "forest.load(\"predictors/sklearn/trained_models/experiment_rf\")\n",
+    "# TODO: I don't think we can possibly load a model this big\n",
+    "# forest.load(\"predictors/sklearn/trained_models/no_overlap_rf\")\n",
```

Comment:
You saved the model a few cells above. So it's not too big to be loaded. Should you remove the TODO?